### PR TITLE
Don't create a new deployment on deploy

### PR DESCRIPTION
**Pull request by stojg**

### CODE REVIEW

File: code/control/DeployDispatcher.php [hunk 1]
```diff
@@ -172,25 +172,31 @@ public function save(\SS_HTTPRequest $request) {
 	 * @return \SS_HTTPResponse
 	 */
 	public function start(SS_HTTPRequest $request) {
+
+		if (strtolower($request->httpMethod()) !== 'post') {
```
Comment (halkyon) [inline code/control/DeployDispatcher.php]: httpMethod is always uppercase, so why bother lowering it?

File: code/control/DeployDispatcher.php [hunk 2]
```diff
@@ -172,25 +172,31 @@ public function save(\SS_HTTPRequest $request) {
 	 * @return \SS_HTTPResponse
 	 */
 	public function start(SS_HTTPRequest $request) {
+
```
Comment (halkyon) [inline code/control/DeployDispatcher.php]: personally I don't like the additions of newlines in the first line of methods... :)

File: code/control/DeployDispatcher.php (hunk 1 above)
Comment (stojg) [inline code/control/DeployDispatcher.php]: well, look at that, I didn't now SS did that. nifty.

**Comment by stojg** [comment]
Feedback dealt with

**Pull request merged**